### DISCUSSION TITLE
How do I set a default pen size?

### DISCUSSION THREAD

**Discussion by mmc-terraform** [pubpub-zz/ppInk#205] [Q&A] [answered]
I would like to have my pen sizes set to 150 by default. The changes I make are only persisting until I close the program. Also invoking a pen through REST-API resets it to 80. 

There is no option on the config file too.

**Comment by pubpub-zz** [comment]
the default values can be modified in the config.ini
```
#increments for penwidth +/- shortcuts
PenWidth_Delta = 40
# defaults for options
PenWidth_Thin_Default = 30
PenWidth_Normal_Default = 80
PenWidth_Thick_Default = 500
```

Can you indicate which REST request is doing the reseting ?

**Reply by mmc-terraform** [reply]
https://user-images.githubusercontent.com/79689065/189522909-343c35df-ac30-4455-b0df-373d6f193c54.mp4

Changing the values on config.ini doesn't work. Could you please tell me if I'm doing something wrong?

**Reply by mmc-terraform** [reply]
The setup on pens.ini works.

**Comment by mmc-terraform** [comment]
Duh! It was there on the config file. Sorry for the stupid question :)

**Comment by mmc-terraform** [comment, marked as answer]
Changed the default size and color on pens.ini instead of config.ini

**Comment by pubpub-zz** [comment]
Sorry for the late answer.
Good news you've fixed it. changing the pen width is the good solution (or typing new value in the options) as it is the numerical value is used.
The parameters I've shown in the config.ini are only used to define the value associated with thin/normal/large in the dropbox in the pen options
